doc/classes/ProjectSettings.xml

```diff
@@ -205,8 +205,6 @@
 		<member name="audio/mix_rate" type="int" setter="" getter="">
 			Mix rate used for audio. In general, it's better to not touch this and leave it to the host operating system.
 		</member>
-		<member name="audio/output_latency" type="int" setter="" getter="">
```

Comment:
That's weird too, all drivers seem to define it. Did you compile without audio support?